wiki/Application-Issues.md

```diff
@@ -1,8 +1,27 @@
+### Electron based applications
+
+Electron based applications can work directly on Wayland, but they don't accept it without configuration. Configuration differs between versions. Information was grabbed from the [ArchWiki](https://wiki.archlinux.org/title/Wayland#Electron).
+
+For Electron (>28) you need to set environment variable:
+```kdl
+environment {
+    ELECTRON_OZONE_PLATFORM_HINT "auto"
+}
+```
+
+For previous version you need to set flags in configuration file `~/.config/electron-flags.conf`:
+```
+--enable-features=UseOzonePlatform
+--ozone-platform-hint=auto
+```
+
 ### VSCode
 
-If you're having issues with some VSCode hotkeys, try starting `Xwayland` and setting the `DISPLAY=:0` environment variable for VSCode.
-That is, still running VSCode with the Wayland backend, but with `DISPLAY` set to a running Xwayland instance.
-Apparently, VSCode currently unconditionally queries the X server for a keymap.
```

Comment:
Why did you remove these lines? They apply to VSCode running natively on Wayland.

Reply:
because if you run VSCode via platform "wayland", then you don't need to start Xwayland and also no issues with hotkeys.

Reply:
I added this in response to someone literally having this exact issue when running VSCode on Wayland..

Reply:
I mean, maybe they fixed the problem since? But I wasn't affected by this so idk

Reply:
Okay, I'll revert

Reply:
Reverted.